packages/backend/src/services/HealthService/HealthService.ts

```diff
@@ -134,7 +134,9 @@ export class HealthService extends BaseService {
                         this.lightdashConfig.auth.pat.maxExpirationTimeInDays,
                 },
                 snowflake: {
-                    enabled: !!this.lightdashConfig.auth.snowflake.clientId,
+                    enabled:
+                        !!this.lightdashConfig.auth.snowflake.clientId &&
+                        !!this.lightdashConfig.license.licenseKey,
```

Comment:
This will hide the option in the UI if there is no license key

If a license key is provided, it needs to be valid, since we check the vailidty on startup 

![image](https://github.com/user-attachments/assets/68e23085-634b-4d05-999d-ac24fc55fd0e)